examples/01_dirty_categories.py

```diff
@@ -384,9 +384,12 @@
 plt.show()
 
 ###############################################################################
-# We can deduce from this data that the three factors that define the
-# most the salary are: being hired for a long time, being a manager, and
-# having a permanent, full-time job :)
+# We can see that features such the time elapsed since being hired, having a
+# full-time employment, and the position, seem to be the most informative for
+# prediction. However, feature importances must not be over-interpreted -- they
+# capture statistical associations `rather than causal effects <https://en.wikipedia.org/wiki/Correlation_does_not_imply_causation>`_.
```

Comment:
```suggestion
# capture statistical associations `rather than causal effects
# <https://en.wikipedia.org/wiki/Correlation_does_not_imply_causation>`_.
```